cross-modal-search/get_data30k.sh

```diff
@@ -1,4 +1,13 @@
+#!/bin/sh
 pip install kaggle
-kaggle datasets download hsankesara/flickr-image-dataset
-unzip flickr-image-dataset.zip
-rm flickr-image-dataset.zip
\ No newline at end of file
+kaggle datasets download hsankesara/flickr-image-dataset && \
+unzip flickr-image-dataset.zip && \
+rm flickr-image-dataset.zip && \
+wget http://www.cs.toronto.edu/~faghri/vsepp/data.tar && \
```

Comment:
It's better to silence the output
```suggestion
wget -q http://www.cs.toronto.edu/~faghri/vsepp/data.tar && \
```

Reply:
29df17585380b7d85eb3bbcda06d4c0f64393062